### PR TITLE
docs: add DragHandle documentation page

### PR DESCRIPTION
**Pull request by massao**

### CODE REVIEW

File: packages/components/drag-handle/DragHandle.mdx
Comment (gui-santos) [inline packages/components/drag-handle/DragHandle.mdx:69]: Could you rebase this branch before merging it?

I made a change to this title in the plop template that is already merged. It's a couple of react components now and it includes a link to storybook page

Check the documentation of Accordion to see what to do, but this is how I think the code needs to be here

```suggestion
<Stack justifyContent="space-between" marginTop="spacing2Xl" marginBottom="spacingL">
  <Heading id="props-api-reference" as="h2" marginTop="none" marginBottom="none">
    Props (API reference)
  </Heading>

  <TextLink
    href="https://v4-f36-storybook.netlify.app/?path=/story/components-draghandle"
    target="_blank"
    alignIcon="end"
    icon={<ExternalLinkIcon />}
  >
    Open in Storybook
  </TextLink>
</Stack>
```

But you will need to import these components in the MDX file or this page will break

**Review by gui-santos — CHANGES_REQUESTED** [review]
Looks great!

I only have one important suggestion to implement the Props section title with a link to storybook

File: packages/components/drag-handle/DragHandle.mdx
Comment (massao) [reply packages/components/drag-handle/DragHandle.mdx:67]: Good catch, seems like the `Card` component has no styling for the drag handle, only the `AssetCard`, and the `EntryCard`, but `Card` still accepts the `withDragHandle` 🤔 
I'll just remove the withDragHandle from the Card for now, and keep on the other twos, so this PR is not blocked by this, and we can check later why it's like this

**Review by gui-santos — APPROVED** [review]

**Review by bgutsol — APPROVED** [review]

**Pull request merged**
